Library/Formula/jflex.rb

```diff
@@ -0,0 +1,18 @@
+class Jflex < Formula
+  desc "Lexical analyzer generator for Java, written in Java."
+  homepage "http://jflex.de/"
+  url "http://jflex.de/release/jflex-1.6.1.zip"
+  sha256 "6da3c573db065f535c9b544e46ab4d49caa629b0354f8340df027f35e3368a51"
+
```

Comment:
Mind chucking a `bottle :unneeded` line here? Should be:

```ruby
url "http://jflex.de/release/jflex-1.6.1.zip"
sha256 "6da3c573db065f535c9b544e46ab4d49caa629b0354f8340df027f35e3368a51"

bottle :unneeded

depends_on :java => "1.7+"
```
In terms of placement.